packages/components/src/components/input-field/increment-buttons.tsx

```diff
@@ -79,7 +81,9 @@ const IncrementButtons = ({
         <React.Fragment>
             <Button
                 id={`${id}_add`}
-                className={'dc-input-wrapper__button dc-input-wrapper__button--increment'}
+                className={`dc-input-wrapper__button dc-input-wrapper__button--increment ${
+                    should_apply_disabled_style ? 'dc-input-wrapper__button--disabled' : ''
```

Comment:
```
className={classNames('dc-input-wrapper__button dc-input-wrapper__button--increment', {
    'dc-input-wrapper__button--disabled': !!max_is_disabled,
})}
```
@kate-deriv we can get rid of should_apply_disabled_style. :)

Reply:
@maryia-deriv they have different values on desktop after you purchase a contract
<img width="935" alt="Screenshot 2023-08-18 at 1 06 38 PM" src="https://github.com/binary-com/deriv-app/assets/121025168/8110554b-ba56-475b-8cb0-aed2fe4c2311">